src/components/elements/Commentary/Commentary.tsx

```diff
@@ -0,0 +1,76 @@
+import { When } from "react-if";
+
+import Text from "@/components/elements/Text/Text";
+export interface ComentaryFilesProps {
+  id: string;
+  file: string;
+}
+export interface ComentaryProps {
+  name: string;
+  lastName: string;
+  date: string;
+  comentary: string;
+  status?: string;
+  files?: ComentaryFilesProps[];
+}
+
+const statusStyle = {
+  Submitted: { container: "bg-primary-200", textColor: "text-primary" },
+  Draft: { container: "bg-pinkCustom-200", textColor: "text-pinkCustom" }
+};
+
+const Comentary = (props: ComentaryProps) => {
+  const { name, lastName, date, comentary, files = [], status } = props;
+  const statusKey = status as keyof typeof statusStyle;
```

Comment:
This type should be in the CommentaryProps interface instead of simply `string`, then you don't need two variables that hold the same value, and consumers of the component know what the valid values are.

Reply:
done